Tests/RxSwiftTests/Anomalies.swift

```diff
@@ -59,10 +59,9 @@ extension AnomaliesTest {
         }
 
         for op in [
-                { $0.shareReplay(1) },
+                { $0.share(replay: 1) },
                 { $0.replay(1).refCount() },
-                { $0.publish().refCount() },
-                { $0.shareReplayLatestWhileConnected() }
```

Comment:
I removed `shareReplayLatestWhileConnected` since its the same as `share(replay: 1)` with default scope